### DISCUSSION TITLE
VMC EXPLOIT I DIDNT SEE ON THE LIST

### DISCUSSION THREAD

**Discussion by Synthacol** [3kh0/ext-remover#854] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
First do ctrl+alt+t and once you are in crosh type in vmc. If there is a list of vmc commands then your good but if not then thsi exploit wont work for you. If the vmc works then you esc+reboot+power then control d and keep pressing enter to clear your local data. Then you sign in like normal but as soon as you put in your password and click enter wait five seconds then turn off you wifi. Then when you are in your chromebook make sure your wifi is still off then do ctrl+alt+t to open crosh. once your are there type in "vmc create-extra-disk --size 1 /home/chronos/user/Extensions/". (Note this will make it so you wont be able to have any extensions installed). If you want to revert this you just have to powerwash your chromebook again.

**Comment by rwg100** [comment]
on a lot of school chromebooks crosh is blocked so womp womp this doesn't work

**Reply by Synthacol** [reply]
Yea i know thats why i listed the requirements 🫥

**Comment by NOoOBBBy** [comment]
Does this actually work?

**Reply by Synthacol** [reply]
Yes as long as you have crosh enabled and when you type in vmc on crosh it gives you a string of commands.

**Comment by racerretrocoder** [comment]
Hi there thanks for this! i love it i would sugest removing it because i think my school knows! this is the only way left
